operations/mimir-mixin/mixin-compiled-gem.libsonnet

```diff
@@ -0,0 +1,26 @@
+(import 'mixin.libsonnet') + {
+  // This config enables features that are relevant to most GEM deployments.
+  _config+:: {
+    gem_enabled: true,
+
+    gateway_enabled: true,
```

Comment:
Should we make `gateway_enabled` default to `true` if `gem_enabled` is `true`?

Reply:
yep; will do in the next PR because gem_enabled isn't actually defined